src/it/it-source-failed/src/test/java/name/valery1707/kaitai/it/KaitaiIcoTest.java

```diff
@@ -0,0 +1,49 @@
+package name.valery1707.kaitai.it;
+
+import io.kaitai.struct.ByteBufferKaitaiStream;
+import io.kaitai.struct.KaitaiStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KaitaiIcoTest {
+	private KaitaiStream getResourceAsStream(String resource) {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.